.github/workflows/python-docs.yml

```diff
@@ -0,0 +1,27 @@
+name: documentation (python)
```

Comment:
why a separate workflow? seems a bit cleaner to add a new job to existing python workflow